scalding-core/src/main/scala/com/twitter/scalding/FileSource.scala

```diff
@@ -142,6 +142,31 @@ object FileSource {
     !glob(globPath, conf, SuccessFileFilter).isEmpty
   }
 
+  def allPathsCommitted(globPath: String, conf: Configuration): Boolean = {
```

Comment:
The use of Committed in scalding here is new, i'd rather it was something related to success files so we keep the naming constant